src/NaturalLanguage/NaturalLanguageClient.php

```diff
@@ -0,0 +1,337 @@
+     * use Google\Cloud\NaturalLanguage\NaturalLanguageClient;
+     *
+     * $naturalLanguage = new NaturalLanguageClient([
+     *     'projectId' => 'my-awesome-project'
+     * ]);
+     * ```
+     *
+     * @param array $config {
+     *     Configuration Options.
+     *
+     *     @type string $projectId The project ID from the Google Developer's
+     *           Console.
+     *     @type callable $authHttpHandler A handler used to deliver Psr7
+     *           requests specifically for authentication.
+     *     @type callable $httpHandler A handler used to deliver Psr7 requests.
+     *     @type string $keyFile The contents of the service account
+     *           credentials .json file retrieved from the Google Developers
+     *           Console.
+     *     @type string $keyFilePath The full path to your service account
+     *           credentials .json file retrieved from the Google Developers
+     *           Console.
+     *     @type int $retries Number of retries for a failed request. Defaults
+     *           to 3.
+     *     @type array $scopes Scopes to be used for the request.
+     * }
+     * @throws \InvalidArgumentException
+     */
+    public function __construct(array $config = [])
+    {
+        if (!isset($config['scopes'])) {
+            $config['scopes'] = [self::FULL_CONTROL_SCOPE];
+        }
+
+        $this->connection = new Rest($this->configureAuthentication($config));
+    }
+
+    /**
+     * Finds named entities (currently finds proper names) in the text, entity
+     * types, salience, mentions for each entity, and other properties in the
+     * document.
+     *
+     * Example:
+     * ```
+     * $document = $language->analyzeEntities('Google Cloud Platform is a powerful tool.');
+     *
+     * foreach ($document->entities() as $entity) {
+     *     echo $entity['type'];
+     * }
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/analyzeEntities Analyze Entities API documentation
+     * @codingStandardsIgnoreEnd
+     *
+     * @param string|Object $content The content to analyze.
+     * @param array $options {
+     *     Configuration options.
+     *
+     *     @type string $type The document type. Acceptable values are
+     *           `PLAIN_TEXT` or `HTML`. Defaults to `PLAIN_TEXT`.
+     *     @type string $language The language of the document. Both ISO
+     *           (e.g., en, es) and BCP-47 (e.g., en-US, es-ES) language codes
+     *           are accepted. Defaults to English.
+     *     @type string $encodingType The text encoding type used by the API to
+     *           calculate offsets. Acceptable values are `NONE`, `UTF8`,
+     *           `UTF16` and `UTF32`. Defaults to `UTF8`.
+     * }
+     * @return Document
+     */
+    public function analyzeEntities($content, array $options = [])
+    {
+        return new Document(
```

Comment:
In these methods where you're only analyzing a single feature, why not return the result of `Document::entities()` instead of requiring the 2nd (superfluous) method call?

In other words, `$nl->analyzeEntities()` > `$nl->analyzeEntities()->entities()`.